lib/matplotlib/tests/test_marker.py

```diff
@@ -204,3 +205,75 @@ def test_marker_clipping(fig_ref, fig_test):
     ax_test.set(xlim=(-0.5, ncol), ylim=(-0.5, 2 * nrow))
     ax_ref.axis('off')
     ax_test.axis('off')
+
+
+@pytest.mark.parametrize("marker,transform,expected", [
+    (markers.MarkerStyle("o"), Affine2D().translate(1, 1),
+        Affine2D().translate(1, 1)),
+    (markers.MarkerStyle("o", transform=Affine2D().translate(1, 1)),
+        Affine2D().translate(1, 1), Affine2D().translate(2, 2)),
+    # (markers.MarkerStyle("$|||$", transform=Affine2D().translate(1, 1)),
+    #  Affine2D().translate(1, 1), Affine2D().translate(2, 2)),
```

Comment:
This should either be used or deleted

Reply:
I cannot use it due to #20943, but I think it should be tested. I hoped that #20921 would be closed at the time when this PR will be ready for merge :-) Sorry for chaining PRs again, but I wanted to split remotely related issues to smaller PRs.